tools/msftidy.rb

```diff
@@ -595,9 +595,21 @@ def check_udp_sock_get
   def check_invalid_url_scheme
     test = @source.scan(/^#.+http\/\/metasploit.com/)
     unless test.empty?
-      test.each { |item|
+      test.each do |item|
         info("Invalid URL: #{item}")
-      }
+      end
+    end
+  end
+
+  # This checks looks for extranous commas
+  # Stuff with comments should match too: , #metasploit module ]
```

Comment:
`, #metasploit module ]` is the whole string that matches. Normaly it searches for `,]` and `,}` but there also comments between them.